routes/olinAuth.js

```diff
@@ -7,7 +7,7 @@ var User = require("../models/user");
 var router = express.Router();
 
 router.get("/login", function(req, res) {
-  res.redirect("http://www.olinapps.com/external?callback="+"http://localhost:3000/olinAuth/auth");
+  res.redirect("http://www.olinapps.com/external?callback="+"http://localhost:3000/olinAuth/auth"+"?req="+req.query.req);
```

Comment:
Wow why didn't we think of this in our OlinJS project? Now I need to implement it!